docs/sample_schema_package/mailman_messages/__init__.py

```diff
@@ -2,7 +2,7 @@
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
+# the Free Software Foundation; either version 3 of the License, or
```

Comment:
This is still changing the license. By the way, if you start using the `SPDX-License-Identifier` tag, you can drop this block of text.